js/src/lib/Details/JSONLDItem.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { packageLink } from '../util';
+
+/**
+ * JSON-ld is a technology that allows crawlers to get info about
```

Comment:
I don't think crawlers will ever see this since it's rendered using JavaScript. We'd need to server render to actually make this useful 😃